apps/user-office-backend/src/eventHandlers/messageBroker.ts

```diff
@@ -145,33 +144,22 @@ export function createPostToRabbitMQHandler() {
         const proposalStatus =
           await proposalSettingsDataSource.getProposalStatus(proposal.statusId);
 
-        // if the new status isn't 'SCHEDULING' ignore the event
```

Comment:
This change will require a small modification in the scheduler messageBroker as well where we handle the events of type: `PROPOSAL_STATUS_CHANGED_BY_WORKFLOW` and `PROPOSAL_STATUS_CHANGED_BY_USER`. I think this is the proper way of working with events where the sender doesn't care for some specific cases and it is just sending the event with all needed information. The receiver is the one that should check if there is some action needed based on the event type or event message.